sdcm/utils/common.py

```diff
@@ -2966,3 +2969,47 @@ def __exit__(self, exit_type, value, _traceback):
 def time_period_str_to_seconds(time_str: str) -> int:
     """Transforms duration string into seconds int. e.g. 1h -> 3600, 1h22m->4920 or 10m->600"""
     return sum([int(g[0] or 0) * 3600 + int(g[1] or 0) * 60 + int(g[2] or 0) for g in duration_pattern.findall(time_str)])
+
+
+def get_keyspace_partition_ranges(node, keyspace: str):
+    result = node.run_nodetool("describering", keyspace)
```

Comment:
I don't understand why you need to run "describering" to get a token.
the token range is a fixed range, the entire range is always:
-9223372036854775807 ≤ token(id) ≤ 9223372036854775807

Reply:
So why not dropping this method?

Reply:
I think it maybe userfull. This method tested and work well. so I decided not remove it